app/src/pages/contact.tsx

```diff
@@ -1,76 +1,74 @@
-import { useAppContext } from '@context/state'
```

Comment:
Notes for reviewers: The Contact page (`/contact`) is the last editable form page in the form wizard flow. The back link routes to `/choose-clinic` and the call to action button routes the user to `/review`. In order for a user to be allowed onto this page, they must have filled out all previous pages with valid data. 

![Screen Shot 2022-10-28 at 2 40 21 AM](https://user-images.githubusercontent.com/67701/198556955-64fece63-5f33-4514-bc7b-5917a731a849.png)